examples/jsm/tsl/display/hashBlur.js

```diff
@@ -10,42 +10,32 @@ import { float, Fn, vec2, uv, sin, rand, degrees, cos, Loop, vec4, premultiplyAl
  * @param {Node<float>} [bluramount=float(0.1)] - This node determines the amount of blur.
  * @param {Object} [options={}] - Additional options for the hash blur effect.
  * @param {Node<float>} [options.repeats=float(45)] - The number of iterations for the blur effect.
- * @param {Node<vec4>} [options.mask=null] - A mask node to control the alpha blending of the blur.
```

Comment:
Are you okay with adding a bit more description to the blur?
```
The appraoch of this blur is different compared to Gaussian and box blur since
it does not rely on a kernel to apply a convolution. Instead, it reads the base
texture multiple times (defined by the `repeats` parameter) in a random pattern and
then averages the samples. A typical artifact of this technique is a slightly
noisy appearance of the blur which can be mitigated by increasing the number of
iterations. Compared to Gaussian blur, hash blur requires just a single pass.
```